pkg/maps/multicast/subscribermap.go

```diff
@@ -124,6 +132,19 @@ func NewGroupV4Map(in ParamsIn) ParamsOut {
 	return out
 }
 
+// CheckBatchLookupSupported checks if the kernel supports batch lookup for the passed map.
+// Note, key and value types for the map must be passed in the generic form.
+func CheckBatchLookupSupported[K, V any](m *ebpf.Map) bool {
```

Comment:
This was made to be generic and is exported, but lives inside our subscriber map package. Is that intended?

Reply:
Not sure if there are other consumers for it. I can move this to probe package ?